lap06.playground/Contents.swift

```diff
@@ -0,0 +1,46 @@
+//1
+
+import Darwin
+import Foundation
+import CoreFoundation
+
+for index in stride (from: 5, through: 0, by: -1){
+    print(index)
+}
+var index = Int.random(in: 1...5)
```

Comment:
@AfafAlqahtani great job for solving the questions 

but for Q1 we needed to sort the array in a random way for example [1,2,3] => [3,1,2]